stage2/modules/http/README.md

```diff
@@ -1,26 +1,83 @@
 
-1.  [HTTP Crash Course & Exploration](https://youtu.be/iYM2zFP3Zn0) - 40 минут
-2.  [Introducing CORS: Cross-Origin Resource Sharing](https://www.youtube.com/watch?v=JVZIhCVFJ9c) - 8 минут
+1. **HTTP:**
+
+- [An overview of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
+- [HTTP Lecture [RU]](https://www.youtube.com/watch?v=jOUb8wot2sU)
+- [HTTP Lecture Slides](https://slides.com/dzmitrytsebruk/http)
+
+2. **HTTPS:**
+
+- [HTTP vs HTTPS – What's the Difference?](https://www.freecodecamp.org/news/http-vs-https/)
+
+3. **CORS:**
+
+- [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
+- [Fetch: Cross-Origin Requests](https://javascript.info/fetch-crossorigin)
+
+## Practice 💻
+
+- Complete the test "[St2] HTTP/HTTPS" in the RS-App > Auto Test.
+
+## Additional Resources 📘
+
+Expand your knowledge with these additional materials:
+
+- **Network Communication:**
+
+  - [RS School Webinar on Network Communication part 1 [RU]](https://www.youtube.com/watch?v=4jA9Nea51T8)
+  - [RS School Webinar on Network Communication part 2 [RU]](https://www.youtube.com/watch?v=_8GoJck9O9Y)
+
+- **HTTP:**
+
+  - [HTTP - CS50](https://www.youtube.com/watch?v=4axL8Gfw2nI)
+  - [HTTP Crash Course & Exploration](https://youtu.be/iYM2zFP3Zn0)
+
+- **CORS:**
+
+  - [Introducing CORS: Cross-Origin Resource Sharing](https://www.youtube.com/watch?v=JVZIhCVFJ9c)
+
+- **API Security Best Practices:**
+
+  - [OWASP Top 10 API Security Risks – 2023](https://owasp.org/API-Security/editions/2023/en/0x11-t10/)
+  - [The Fundamentals of Rate Limiting](https://medium.com/@patrikkaura/the-fundamentals-of-rate-limiting-how-it-works-and-why-you-need-it-fd86d39e358d)
+  - [Input Validation Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html)
+  - [Error Handling Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Error_Handling_Cheat_Sheet.html)
+
+- **Authentication and Authorization:**
+
+  - [Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization)
+  - [Authentication vs Authorization](https://www.freecodecamp.org/news/whats-the-difference-between-authentication-and-authorisation/)
+  - [Authorization Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Authorization_Cheat_Sheet.html)
+  - [Authentication Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html)
+  - [OAuth 2 Simplified](https://aaronparecki.com/oauth-2-simplified/)
+  - [Tutorial: How OAuth 2 works with real world example](https://www.youtube.com/watch?v=RROVLdfGmuE)
+  - [JSON Web Tokens (JWT) — the only explanation you will ever need](https://arielweinberger.medium.com/json-web-token-jwt-the-only-explanation-youll-ever-need-cf53f0822f50)
+
+- **Emerging Technologies and Future Trends:**
+
+  - [HTTP/3 From A To Z: Core Concepts](https://www.smashingmagazine.com/2021/08/http3-core-concepts-part1/)
```

Comment:
We already suggested studying this article as part of the module
https://github.com/rolling-scopes-school/tasks/tree/master/stage1/modules/client-server